harness/determined/deploy/gke/cli.py

```diff
@@ -0,0 +1,551 @@
+            "--scopes=storage-full,cloud-platform",
+            "--node-labels=accelerator_type=cpu",
+            "--node-taints=gpuAvailable=False:NoSchedule",
+        ]
+        subprocess.check_call(cmd)
+
+
+def configure_helm(args: argparse.Namespace) -> None:
+    helm_dir = Path(args.helm_dir)
+    with (helm_dir / "Chart.yaml").open() as f:
+        helm_chart = safe_load_yaml_with_exceptions(f)
+    if args.det_version:
+        helm_chart["appVersion"] = args.det_version
+    elif "dev" in helm_chart["appVersion"]:
+        # Preserve user overridden appVersion in helm chart unless it includes dev in the version.
+        helm_chart["appVersion"] = determined.__version__
+    if args.gpu_coscheduler:
+        helm_chart["defaultScheduler"] = "coscheduler"
+    elif args.preemption:
+        helm_chart["defaultScheduler"] = "preemption"
+    with (helm_dir / "Chart.yaml").open("w") as f:
+        yaml.round_trip_dump(helm_chart, f)
+    with (helm_dir / "values.yaml").open() as f:
+        helm_values = safe_load_yaml_with_exceptions(f)
+    checkpointStorage = {}
+    checkpointStorage["saveExperimentBest"] = helm_values["checkpointStorage"].get(
+        "saveExperimentBest", 0
+    )
+    checkpointStorage["saveTrialBest"] = helm_values["checkpointStorage"].get("saveTrialBest", 1)
+    checkpointStorage["saveTrialLatest"] = helm_values["checkpointStorage"].get(
+        "saveTrialLatest", 1
+    )
+    checkpointStorage["type"] = "gcs"
+    checkpointStorage["bucket"] = args.gcs_bucket
+    helm_values["checkpointStorage"] = checkpointStorage
+    helm_values["maxSlotsPerPod"] = args.gpus_per_node
+    if args.multi_node_pool:
+        gpu_pod_spec = make_spec(helm_values["taskContainerDefaults"], "gpuPodSpec")
+        gpu_pod_spec["spec"]["tolerations"] = [
+            {
+                "key": "gpuAvailable",
+                "operator": "Equal",
+                "value": "True",
+                "effect": "NoSchedule",
+            }
+        ]
+        gpu_pod_spec["spec"]["nodeSelector"] = {"accelerator_type": "gpu"}
+        helm_values["taskContainerDefaults"]["gpuPodSpec"] = gpu_pod_spec
+        cpu_pod_spec = make_spec(helm_values["taskContainerDefaults"], "cpuPodSpec")
+        cpu_pod_spec["spec"]["tolerations"] = [
+            {
+                "key": "gpuAvailable",
+                "operator": "Equal",
+                "value": "False",
+                "effect": "NoSchedule",
+            }
+        ]
+        cpu_pod_spec["spec"]["nodeSelector"] = {"accelerator_type": "cpu"}
+        if args.cpu_coscheduler:
+            cpu_pod_spec["spec"]["schedulerName"] = "coscheduler"
+        helm_values["taskContainerDefaults"]["cpuPodSpec"] = cpu_pod_spec
+    with (helm_dir / "values.yaml").open("w") as f:
+        yaml.round_trip_dump(helm_values, f)
+
+
+def handle_up(args: argparse.Namespace) -> None:
+    try:
+        validate_args(args)
+    except Exception as e:
```

Comment:
perhaps be more specific and only catch the type you throw, `ValueError`/`ArgumentError`?